### ISSUE TITLE
Get rid of timeout between the keydown and keyup events

### ISSUE THREAD

**Issue by unnamed777** [Hammerspoon/hammerspoon#1080]
Update 0.9.49 brings this change:

> hs.eventtap.keyStroke() and .keyStrokes() now wait 200ms between the keydown and keyup events, to improve reliability

How can I get rid of it? I use keyStroke() to emulate vi-navigation:
```
hs.hotkey.bind('cmd', "H",
    function () hs.eventtap.keyStroke({}, "Left") end,
    nil,
    function () hs.eventtap.keyStroke({}, "Left") end
);

```

And now I have a huge lag while I'm pressing my bindings.

**Comment by asmagill** [comment]
Check out https://github.com/Hammerspoon/hammerspoon/issues/1011#issuecomment-261114434

I happened upon that thread before I came across this one.

**Comment by cmsj** [comment]
This has also been addressed by 00cf227 which will make the delays configurable.
